cmd/manager/config.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"github.com/spf13/pflag"
+)
+
+const (
+	// Change below variables to serve metrics on different host or port.
+	defaultTmpDir = "/tmp"
+
+	//Kops
+	defaultKopsStateStore     = "s3://kops.state.seizadi.infoblox.com"
+	defaultKopsClusterDnsZone = "soheil.belamaric.com"
```

Comment:
This variable wasn't used until now, so I will have to update the helm chart to include this env variable. This is me just making a mental note.

Reply:
Resolved in #51